### ISSUE TITLE
Angulated muscle fibers

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7170]
**Preferred term label:**
Angulated muscle fibers

**Synonyms**
Angular muscle fibers

**Definition (free text, please give PubMed ID)**

Def: Normal muscle fibers are polygonal-shaped in cross section, are multinucleated, and have minimal amounts of endomysial connective tissue. In contrast, angulated (also known as angular) muscle fibers have long and narrow vertices (corners) with sharp edges and a pointed tip.
PMID: 22938878

Comment: Muscle fibers showing neurogenic atrophy are compressed by neighboring non-atrophic fibers and may show an angulated appearance on a transverse section. 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619477    PMID:32467133   1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
